src/ripple/protocol/impl/SField.cpp

```diff
@@ -401,4 +401,17 @@ SField::getField (std::string const& fieldName)
     return sfInvalid;
 }
 
+Json::Value SField::allFieldsJson ()
+{
+    Json::Value all(Json::arrayValue);
+    for (auto const& pair : knownCodeToField)
+    {
+        if (pair.second->isBinary())
```

Comment:
This kind of makes the 'isBinary' output pointless ... 

The reason it was there in the first place was that that there's some Hash256 fields like `index` and `hash` that are useful to declare, such that any Json -> dehydrated class works simply.  There's also other Amount fields like "taker_gets_funded" etc in offers json.